system/web/services/ModuleService.cfc

```diff
@@ -912,10 +912,13 @@ component extends="coldbox.system.web.services.BaseService" {
 	 * Unload all registered modules
 	 */
 	ModuleService function unloadAll(){
-		// Unload all modules
-		variables.registeredModules.each( function( key, module ){
-			unload( arguments.key );
-		} );
+		// If the registeredModules key exists, and isn't the default empty string
+		if ( structKeyExists( variables, "registeredModules" ) && isStruct( variables.registeredModules ) ) {
```

Comment:
I would avoid the structkeyexists and use `isNull()` to comply with full null support.